src/AcceptanceTests/Features/CalculateEarningsForLearningPayments.feature

```diff
@@ -0,0 +1,5 @@
+﻿Feature: Calculate earnings for learning payments
+
+Scenario: Simple Earnings Generation
+	Given An apprenticeship learner event comes in from approvals
+	Then An earnings generated event is published with the correct learning amounts
```

Comment:
these steps should describe a business requirement / behaviour and be less "techy".

Reply:
sorted